README.md

```diff
@@ -146,6 +147,35 @@ This will store images by default in your document folder, however you may also
 :imagesdir: media
 ```
 
+### Use asciidoctor.js extensions in workspace
+
+This extension supports registration of asciidoctor.js extensions.
+To see the rendered result in the preview, the extension can use asciidoctor.js extensions while rendering it.
+
+To create an autoloadable extension you should do the next steps:
+
+* Enable the feature by `Change AsciiDoc extension scripts Security Settings` command.
+
+**Warning**
+This feature will automatically executes scripts in workspace. So, this feature is disabled by default.
+Enable the feature only if you trust the workspace project.
+
+* Install npm package in the workspace directory. For example, the installation command is the following:
+
+```shell
+npm install asciidoctor-emoji
+```
+
+* This extension will search the directory `.asciidoctor/lib` in the workspace directory and load all files with the extension `js` as Asciidoctor.js extensions. For example, `.asciidoctor/lib/emoji.js` containing the following:
+
+```javascript
+const emoji = require('asciidoctor-emoji')
+
+module.exports.register = function register(registry) {
+    emoji.register(registry)
+}
```

Comment:
Since `require('asciidoctor-emoji')` exports a function named `register` it can be simplified to:

```js
module.exports = require('asciidoctor-emoji')
```

Reply:
It is done.